lib/bolt/config.rb

```diff
@@ -115,17 +115,17 @@ def update_from_file(data)
         if data['winrm']['tmpdir']
           self[:transports][:winrm][:tmpdir] = data['winrm']['tmpdir']
         end
-        if data['winrm']['ca-cert']
-          self[:transports][:winrm][:ca_cert] = data['winrm']['ca-cert']
+        if data['winrm']['cacert']
+          self[:transports][:winrm][:ca_cert] = data['winrm']['cacert']
```

Comment:
can we change this to cacert instead of ca_cert in the config object too. ditto for orch